spec/conformance/mds_finder.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'fido_metadata'
+
+module WebAuthn
+  class MDSFinder
+    extend Forwardable
+
+    def_delegator :fido_metadata_configuration, :cache_backend, :cache_backend
+    def_delegator :fido_metadata_configuration, :cache_backend=, :cache_backend=
+    def_delegator :fido_metadata_configuration, :metadata_token, :token
+    def_delegator :fido_metadata_configuration, :metadata_token=, :token=
+
+    def find(_format, aaguid: nil, attestation_certificate_key_id: nil)
+      metadata_statement =
+        begin
+          if aaguid
+            fido_metadata_store.fetch_statement(aaguid: aaguid)
+          else
+            fido_metadata_store.fetch_statement(attestation_certificate_key_id: attestation_certificate_key_id)
+          end
+        end
```

Comment:
Isn't `begin; end`  redundant here?

Reply:
yes!  It used to go along with a `rescue` but I forgot to remove it 😅

Reply:
Gotcha, hehe.